### PR TITLE
BOM-1288

### PR DESCRIPTION
**Pull request by awais786**
Removed django11 from tox and travis.

### CODE REVIEW

**Comment by awais786** [comment]
There is no setup.py.

**Review by Ayub-Khan — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,6 @@ python:
 - 3.5
 
 env:
-- TOXENV=django111
 - TOXENV=django20
 - TOXENV=django21
```
Comment (jmbowman) [inline .travis.yml]: We should remove the 2.0 and 2.1 tests also, since we're already on 2.2.

Comment (awais786) [reply .travis.yml]: I think we should keep the tests as edx-platform is still on django1.11. So we accedently do not break any django functionality which is not backwords compateble ?

Comment (awais786) [reply .travis.yml]: @jmbowman ^^

Comment (jmbowman) [reply .travis.yml]: xqueue isn't a dependency of edx-platform, it's a separate service which is already running in production on Django 2.2.  So if there were going to be any inter-service communication issues based on the Django version in use, we should have seen that when we did the deployment, but it went smoothly.

**Comment by waheedahmed** [comment]
Unit tests stopped running in travis builds after this change, fixed it in https://github.com/edx/xqueue/pull/723.

CC: @fysheets

**Pull request merged**